src/API/HealthController.ts

```diff
@@ -30,22 +52,60 @@ export class HealthController {
     }
   }
 
-  private async checkIPFS(): Promise<boolean> {
+  private async checkIPFS(): Promise<any> {
     try {
-      const ipfsConnection = await this.ipfs.getVersion()
-      return isStatus200(ipfsConnection)
+      const { ipfsIsConnected } = (await this.collection.findOne({ name: 'ipfsConnected' })) || {
```

Comment:
Please put all mongo logic in a DAO, see StorageWriter. Same goes for all `this.collection` calls.

Reply:
Ok. Is this something I should do in another PR then?

Reply:
Yup, I was just pointing it out so you are aware for next time 👍